README.md

```diff
@@ -1,5 +1,25 @@
 # [`biomejs.dev`](https://biomejs.dev/)
 
+The website source of [biomejs/biome](https://github.com/biomejs/biome).
+
+## Clone
+
+This repository includes a submodule pointing to the [main repository](https://github.com/biomejs/biome), which is required to build wasm artifacts for the [website playground](https://biomejs.dev/).
```

Comment:
The main repository is needed to build the rules pages too. Not yet implemented, but it will